src/modules/counterComment.js

```diff
@@ -0,0 +1,9 @@
+const counterComments = (array) => {
+  let num = 0;
+  if (array.length > 0) {
+    num = array.length;
+  }
```

Comment:
- [ ] Since you pass either `[]` or `item` array here. Wouldn't it work if you just do the following?

```javascript
const counterComments = (array) => {
  const num = array.length;
}
```
[OPTIONAL]